frontend/src/model/beta/releases/ModelReleaseReviewDisplay.tsx

```diff
@@ -0,0 +1,54 @@
+import Done from '@mui/icons-material/Done'
+import HourglassEmpty from '@mui/icons-material/HourglassEmpty'
+import { Stack, Typography } from '@mui/material'
+
+import { useGetModelRoles } from '../../../../actions/model'
+import { ReviewRequestInterface } from '../../../../types/interfaces'
+import { ApprovalStates } from '../../../../types/v2/enums'
+import { getRoleDisplay } from '../../../../utils/beta/roles'
+import Loading from '../../../common/Loading'
+import MessageAlert from '../../../MessageAlert'
+
+interface ModelReleaseReviewDisplayProps {
+  review: ReviewRequestInterface
+}
+
+export default function ModelReleaseReviewDisplay({ review }: ModelReleaseReviewDisplayProps) {
+  const { modelRoles, isModelRolesLoading, isModelRolesError } = useGetModelRoles(review.model.id)
+
+  if (isModelRolesError) {
+    return <MessageAlert message={isModelRolesError.info.message} severity='error' />
+  }
+  return (
+    <>
+      {isModelRolesLoading && <Loading />}
+      <Stack direction={{ sm: 'row', xs: 'column' }}>
+        {review.responses.map((response) => {
+          const [_kind, username] = response.user.split(':')
+          switch (response.decision) {
+            case ApprovalStates.Accepted:
+              return (
+                <Stack direction='row' spacing={1}>
+                  <Done color='success' fontSize='small' />
+                  <Typography variant='caption'>
+                    <span style={{ fontWeight: 'bold' }}>{username}</span>
```

Comment:
Imo we should stick to using `sx` as we do for the rest of the app. Consider using a `<Box component='span' sx={{ 
fontWeight: 'bold' }}>` instead